components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1118,6 +1120,99 @@ public OAuthRevocationResponseDTO revokeAuthzForAppsByResourceOwner(
         return new OAuthRevocationResponseDTO();
     }
 
+    /**
+     * Revoke issued tokens for the application.
+     *
+     * @param application {@link OAuthAppRevocationRequestDTO}
+     * @return revokeRespDTO {@link OAuthAppRevocationRequestDTO}
+     * @throws IdentityOAuthAdminException Error while revoking the issued tokens.
+     */
+    public OAuthRevocationResponseDTO revokeIssuedTokensByApplication(OAuthAppRevocationRequestDTO application)
+            throws IdentityOAuthAdminException {
+
+        triggerPreApplicationTokenRevokeListeners(application);
+        String consumerKey = application.getConsumerKey();
+        String applicationName = application.getApplicationName();
+        String tenantDomain = application.getTenantDomain();
+        if (StringUtils.isNotBlank(consumerKey) && StringUtils.isNotBlank(applicationName)
+                && StringUtils.isNotBlank(tenantDomain)) {
+            List<AccessTokenDO> activeDetailedTokens;
+            try {
+                //Get active access tokens by application id
```

Comment:
```suggestion
                // Get active access tokens by application id.
```
Please make sure all inline comments are formatted as in the given suggestion.

Reply:
Addressed in [6918b82](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/6918b828f239ba766e20f66e6d0af997d891413a)